plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/server/DatastoreIndexUpdateDataTest.java

```diff
@@ -148,5 +161,29 @@ private void createDatastoreIndexesAutoXml() throws IOException {
     File datastoreIndexesAutoXml = new File(appengineGenerated, "datastore-indexes-auto.xml");
     assertTrue(appengineGenerated.mkdirs());
     assertTrue(datastoreIndexesAutoXml.createNewFile());
+    
+    try (Writer out = new OutputStreamWriter(new FileOutputStream(datastoreIndexesAutoXml),
```

Comment:
I remember someone told me to prefer `Files.newOutputStream()`.

Reply:
done